website/docs/r/cloud_project_user_s3_credential.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_cloud_project_user_s3_credential"
+sidebar_current: "docs-ovh-resource-cloud-project-user-s3-credential"
+description: |-
+  Creates an S3 Credential for a user in a public cloud project.
+---
+
+# ovh_cloud_project_user_s3_credential
+
+Creates an S3 Credential for a user in a public cloud project.
+
+## Example Usage
+
+```hcl
+resource "ovh_cloud_project_user" "user" {
+ service_name = "XXX
+ description  = "my user for acceptance tests"
```

Comment:
In order to help user about how they can create this resource, adding a role_name can be useful.

For example:
```
role_name = "administrator"
```

or specify a list of role_names for the example